cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -393,4 +393,26 @@ Resources:
       Strategy: cluster
   {%- endif %}
 {%- endfor %}
+  DynamoDBTable:
+    Type: AWS::DynamoDB::Table
+    UpdateReplacePolicy: Retain
+    DeletionPolicy: Delete
+    Properties:
+      TableName: !Ref 'MainStackName'
+      AttributeDefinitions:
+        - AttributeName: Id
```

Comment:
What about renaming it to Hostname? or something like this?

Reply:
I used `id` only because this table also contains the compute fleet overall status with an item having key `COMPUTE_FLEET`